exercises/E-strings-concatenation/README.md

```diff
@@ -1,12 +1,12 @@
 You can add two strings together using the plus operator (`+`):
 
 ```js
-var greetingStart = "Hello, my name is ";
-var name = "Daniel";
```

Comment:
Hi Lawrence, according to the guidelines, we should try and use "let" instead of "var" - I also used var, so I probably will need to start using Let